projects/sandbox/export/export/main.py

```diff
@@ -174,7 +193,6 @@ def main(
             fduration=fduration,
             fftlength=fftlength,
             highpass=highpass,
-            name="snapshotter",
```

Comment:
was there a reason to delete this? I guess this is probably the default name, but still worth making it clear here what's happening

Reply:
Ah ok nvm seeing that it doesn't get used, nevermind then